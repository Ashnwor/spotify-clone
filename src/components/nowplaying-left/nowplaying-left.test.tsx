import React from 'react';
import { shallow } from 'enzyme';
import NowPlayingLeft from './nowplaying-left';

it('runs correctly', () => {
  const wrapper = shallow(<NowPlayingLeft />);
  expect(wrapper).toMatchSnapshot();
});
