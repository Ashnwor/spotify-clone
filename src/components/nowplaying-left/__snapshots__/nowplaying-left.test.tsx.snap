// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`runs correctly 1`] = `
<div
  className="nowplaying-left-container"
/>
`;
